events/forms.py

```diff
@@ -1,9 +1,14 @@
 from django.forms import ModelForm
-from .models import EventParticipant
+from .models import EventParticipant, requestEvent
 
 
 class ParticipantForm(ModelForm):
     class Meta:
         model = EventParticipant
         fields = ['title', 'student_name',
                   'email_id', 'mobile_number', 'roll_no', 'branch']
+
+class requestEventForm(ModelForm):
```

Comment:
Issues here :
- You're not following PEP8 conventions here.
- According to PEP8 there should be 2 blank lines after a class.
- Whenever you create class, it should be in caps. eg. `class RequestEventForm(ModelForm)`
- Also try to include docstrings in your class/methods.